docs/I18N-CONVERSION-CHECKLIST.md

```
@@ -0,0 +1,306 @@
+   convention. Check `global.*` keys first — avoid creating duplicates.
+2. Add the key to **all five locale files** in a single commit:
+   - `oscarResources_en.properties` — with the English value
+   - `oscarResources_es.properties` — English value + `# TODO: translate`
+   - `oscarResources_fr.properties` — English value + `# TODO: translate`
+   - `oscarResources_pl.properties` — English value + `# TODO: translate`
+   - `oscarResources_pt_BR.properties` — English value + `# TODO: translate`
+3. Values containing non-ASCII characters must use `\uXXXX` Unicode escapes.
+
+```properties
+# oscarResources_en.properties
+admin.configureFax.title=Configure Fax Settings
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)